### PR TITLE
Creating target directory if missing (it can happen when it's a paren…

### PR DESCRIPTION
**Pull request by jcayetano**
When running `promote:make-promotable` in a Maven project with `<packaging>pom</packaging>`, it fails with the following error:

> [ERROR] Failed to execute goal se.jiderhamn:promote-maven-plugin:1.0.0:make-promotable (default) on project mag-parent-pom: Error writing artifacts to file: /Users/XXX/dev/myproject-parent-pom/target/promotable-artifacts.properties (No such file or directory) -> [Help 1]

### CODE REVIEW

File: src/main/java/se/jiderhamn/promote/maven/plugin/PromoteUtils.java [hunk 1]
```diff
@@ -129,6 +129,9 @@ static URI getTargetURI(MavenProject project) {
   static void writeProperties(Log log, MavenProject project, Properties properties) throws MojoExecutionException {
     try {
       File file = getPromotePropertiesFile(project);
+      if (file.getParentFile().mkdirs()) {
+          log.debug("Created missing target directory.");
```
Comment (mjiderhamn) [inline src/main/java/se/jiderhamn/promote/maven/plugin/PromoteUtils.java]: Would you mind using 2 spaces indentation to be consistent with the rest of the code?

**Comment by jcayetano** [comment]
fixed.

**Comment by mjiderhamn** [comment]
Thanks for the Pull Request!

**Comment by jcayetano** [comment]
No worries, any plans to release a 1.0.1?

**Comment by mjiderhamn** [comment]
I had plans for a 1.1.0 with another feature, but since I'm unsure if I'll find the time this week I just made a 1.0.1 release. Should be available in Maven central shortly.

**Pull request merged**